app/models/edition_lead_image.rb

```diff
@@ -0,0 +1,4 @@
+class EditionLeadImage < ApplicationRecord
+  belongs_to :edition
+  belongs_to :image
```

Comment:
I would argue that the EditionLeadImage has one Image. But I wouldn't argue it very forcefully

Reply:
yea agreed, that makes more sense to me 👍

Reply:
We can do something like `  has_one :edition_lead_image, ->(i) { EditionLeadImage.where(image_id: i.id) }, dependent: :destroy` i guess?

Reply:
Hmm maybe leave it as it is - I'd say it's not worth the hassle